### DISCUSSION TITLE
Commerical RISC-V CPU Selling

### DISCUSSION THREAD

**Discussion by JayWuuuuu** [community/community#129695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Dear

Here’s Jay from Sophgo technology which is based in China and provides general computing power for AI.

We have our self-developed RISC-V CPU and TPU and related products based on that.(AI Server,AI BOX, AI Module,AI Acceleration card)

If your company is looking for those kind of products pls let me know.

**Comment by Akash1134** [comment]
Hi @JayWuuuuu , we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
